Dockerfile

```diff
@@ -6,7 +6,7 @@ COPY . /go/src/github.com/Percona-Lab/clickhouse_exporter
 
 WORKDIR /go/src/github.com/Percona-Lab/clickhouse_exporter
 
-RUN make init && make
+RUN make init && make -j4 > /dev/null
```

Comment:
Thanks a lot for contributing
could you remove this change from PR?
it will be better to see details when something will wrong during `docker build`